server/plugin.go

```diff
@@ -85,6 +85,10 @@ func (p *Plugin) OnActivate() error {
 		return errors.WithMessage(err, "OnActivate: failed to register command")
 	}
 
+	if err = p.API.RegisterMobileTrigger(getTrigger()); err != nil {
+		return errors.WithMessage(err, "OnActivate: failed to register mobile trigger")
+	}
```

Comment:
Following the model to support all clients, this would be the only registration the plugin would need to do for the channel header and removing the need to register it on the webapp side - unifying the behavior for both.

Reply:
Exactly.